src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
I can't make sense of that. This check is refining the check from 'did I get a 200' to 'did I get a 200 with a json body with an update_seq in it' in order to crash if source or target is not couchdb. It has nothing to do with whether checkpoints would work.